dianna/dashboard/pages/Time_series.py

```diff
@@ -88,16 +147,24 @@
     for col, method in zip(columns, methods):
         kwargs = method_params[method].copy()
         kwargs['labels'] = [index]
+        if load_example == "Scientific case: FRB":
+            kwargs['_preprocess_function'] = preprocess
 
         func = explain_ts_dispatcher[method]
 
         with col:
             with st.spinner(f'Running {method}'):
-                explanation = func(serialized_model, ts_data=ts_data, **kwargs)
+                explanation = func(serialized_model, ts_data=ts_data_explainer, **kwargs)
 
-            segments = _convert_to_segments(explanation)
+            if load_example == "Scientific case: FRB":
+                # normalize FRB data and get rid of last dimension
+                fig, _ = plot_image(explanation[0, :, ::-1].T,
+                                    ((ts_data + np.min(ts_data)) / (np.max(ts_data) + np.min(ts_data)))[::-1]
+                                    )
```

Comment:
Is the normalization done to scale the range of `ts_data` to [0,1]? if so, the equation should be `(value - min_value) / (max_value - min_value)`

Reply:
one more comment: plotting the original data in this example doesn't contribute any new information and only makes the figure darker. I suggest removing it.